MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2314,6 +2324,11 @@ public Set<PartInUse> getPartsInUse(boolean ignoreMothballedUnits,
             if (inUse.containsKey(partInUse)) {
                 partInUse = inUse.get(partInUse);
             } else {
+                if(piuStockMap.containsKey(partInUse.getDescription())) {
```

Comment:
Can we come up with a better variable name than `piuStockMap` maybe `partsInUseStockMap`, but I'm sure you can come up with something better. I get that this is annoying, but I've lost a lot of time trying to decipher single letter or abbreviated variable names that were probably super obvious when the author wrote them, but a decade later have lost all meaning.

Reply:
this has been changed to partInUseRequestedStockMap, it's long winded but it's a decent descriptor of what's otherwise a difficult to describe concept. The comment above it'd declaration should help somewhat.